src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/ColorPickerViewModel.cs

```diff
@@ -113,10 +117,15 @@ public int CopiedColorRepresentationIndex
             }
         }
 
+        public string GetSettingsSubPath()
+        {
+            return settingsConfigFileFolder + "\\" + ColorPickerSettings.ModuleName;
+        }
+
         private void NotifySettingsChanged()
         {
             SendConfigMSG(
-                   string.Format("{{ \"powertoys\": {{ \"{0}\": {1} }} }}", ColorPickerSettings.ModuleName, JsonSerializer.Serialize(_colorPickerSettings)));
+                   string.Format("{{ \"powertoys\": {{ \"{0}\": {1} }} }}", GetSettingsSubPath(), JsonSerializer.Serialize(_colorPickerSettings)));
```

Comment:
Doesn't this break IPC communication?  Are you sure color picker changes are being saved?

Reply:
i reverted the changes